app.rb

```diff
@@ -0,0 +1,168 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class App
+  def list_books
+    if @books.length.zero?
+      puts "There's no book, Please add a book first"
+    else
+      @books.each_with_index do |book, index|
+        puts "#{index + 1} - Book title: #{book.title}, Book author: #{book.author}"
+      end
+    end
+  end
+
+  def list_persons
+    if @persons.length.zero?
+      puts "There's no person, Please add a person first"
+    else
+      @persons.each_with_index do |person, index|
+        if person.is_a?(Teacher)
+          puts "[Teacher] #{index + 1} - ID: #{person.id}, Name: #{person.name}, Age: #{person.age}"
+        else
+          puts "[Student] #{index + 1} - ID: #{person.id}, Name: #{person.name}, Age: #{person.age}"
+        end
+      end
+    end
+  end
+
+  def person_object(id)
+    @persons.each do |person|
+      return person if person.id == id
+    end
+    nil
+  end
+
+  def list_rentals
```

Comment:
- [ ] Your application is failing in an attempt to list a rental for a given ID. Your application throws an error when details when option 6 is selected for the person who has created a rental. Kindly ensure to improve upon this aspect.

**Error:**
![image](https://user-images.githubusercontent.com/12745474/168404464-c161947d-5797-44bf-adf5-275fcf0c163d.png)